### ISSUE TITLE
Uncaught Error: [$injector:modulerr] when importing 'swaggerUi'

### ISSUE THREAD

**Issue by Audace** [Orange-OpenSource/angular-swagger-ui#45]
When I add swaggerUi to angular module, I get the $injector:modulerr error in Chrome dev console. My import statement is the following:

`var app = angular.module('flapperNews', ['ui.router','elasticsearch','swaggerUi'],['$locationProvider', function($locationProvider) {
  $locationProvider.html5Mode({
    enabled: true
  });
}]).config(function($provide) {
    $provide.decorator('$sniffer', function($delegate) {
      $delegate.history = false;
      return $delegate;
    });
});`

I've run `bower install angular-swagger-ui` and have included it in the `<head>` tag of my index.html file. Has anyone run into this issue? I don't know what more I can do.

**Comment by mathieuales** [comment]
Hi !

Make sure to include script after `angular.js`
Please paste error message and your code included in `head`.

**Comment by Audace** [comment]
Here's the code included in head after your suggestion:

`  <head>

```
<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css"
      rel="stylesheet">
<link href="bower_components/angular-swagger-ui/dist/css/swagger-ui.min.css"
      rel="stylesheet">
<link href="bower_components/ng-prettyjson/dist/ng-prettyjson.min.css" rel="stylesheet">
<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
<script src="http://ajax.googleapis.com/ajax/libs/angularjs/1.3.10/angular.min.js"></script>
<script src="bower_components/angular-swagger-ui/dist/scripts/swagger-ui.min.js"></script>
<script src="http://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.10/angular-ui-router.js"></script>
<script src="bower_components/swagger-parser/dist/swagger-parser.js"></script>
<script src="bower_components/tg-angular-validator/dist/angular-validator.js"></script>
<script src="bower_components/elasticsearch/elasticsearch.angular.js"></script>
<script src="bower_components/angular-sanitize/angular-sanitize.min.js"></script>
<script src="bower_components/ng-prettyjson/dist/ng-prettyjson.min.js
"></script>
<script src="/javascripts/angularApp.js"></script>
<style> .glyphicon-thumbs-up { cursor:pointer } </style>
<base href="/" />
```

  </head>`

I still receive the following error:
`angular.js:38
Uncaught Error: [$injector:modulerr] http://errors.angularjs.org/1.3.10/$injector/modulerr?p0=flapperNews&p1=Err...gleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js%3A17%3A350)`

Clicking on the link takes me here: https://docs.angularjs.org/error/$injector/modulerr?p0=flapperNews&p1=Error:%20%5B$injector:modulerr%5D%20http:%2F%2Ferrors.angularjs.org%2F1.3.10%2F$injector%2Fmodulerr%3Fp0%3DswaggerUi%26p1%3DTypeError%253A%2520angular.merge%2520is%2520not%2520a%2520function%250A%2520%2520%2520%2520at%2520addTranslations%2520(http%253A%252F%252Flocalhost%253A3000%252Fbower_components%252Fangular-swagger-ui%252Fdist%252Fscripts%252Fswagger-ui.min.js%253A7%253A11281)%250A%2520%2520%2520%2520at%2520http%253A%252F%252Flocalhost%253A3000%252Fbower_components%252Fangular-swagger-ui%252Fdist%252Fscripts%252Fswagger-ui.min.js%253A7%253A12182%250A%2520%2520%2520%2520at%2520Object.e%2520%255Bas%2520invoke%255D%2520(http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A37%253A96)%250A%2520%2520%2520%2520at%2520d%2520(http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A35%253A301)%250A%2520%2520%2520%2520at%2520http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A35%253A425%250A%2520%2520%2520%2520at%2520s%2520(http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A7%253A302)%250A%2520%2520%2520%2520at%2520g%2520(http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A35%253A202)%250A%2520%2520%2520%2520at%2520http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A35%253A371%250A%2520%2520%2520%2520at%2520s%2520(http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A7%253A302)%250A%2520%2520%2520%2520at%2520g%2520(http%253A%252F%252Fajax.googleapis.com%252Fajax%252Flibs%252Fangularjs%252F1.3.10%252Fangular.min.js%253A35%253A202)%0A%20%20%20%20at%20Error%20(native)%0A%20%20%20%20at%20http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:6:417%0A%20%20%20%20at%20http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:36:98%0A%20%20%20%20at%20s%20(http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:7:302)%0A%20%20%20%20at%20g%20(http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:35:202)%0A%20%20%20%20at%20http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:35:371%0A%20%20%20%20at%20s%20(http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:7:302)%0A%20%20%20%20at%20g%20(http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:35:202)%0A%20%20%20%20at%20Ob%20(http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:38:435)%0A%20%20%20%20at%20d%20(http:%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fangularjs%2F1.3.10%2Fangular.min.js:17:350

**Comment by Audace** [comment]
It looks like if you read the text of the url, it's saying the angular.merge is not a function in addTranslations. Do I have an outdated version of Angular or is this an issue?

**Comment by mathieuales** [comment]
Yes `angular.merge` is only available in version >= 1.4.0
